lib/fluentReports.js

```diff
@@ -3426,25 +3426,33 @@
             if (options.dash) { this._PDF.undash(); }
         },
 
-        _truncateText: function (data, width) {
-            var curData;
-            var offset = data.data.indexOf('\n');
-            if (offset > 0) {
-                curData = data.data.substr(0, offset);
-            } else {
-                curData = data.data;
-            }
-            var wos = this.widthOfString(curData);
+        _truncateText: function (data, width, lineData) {
```

Comment:
Can we eliminate the lineData -- it appears that you added it; but don't need it now.